evap/staff/tools.py

```diff
@@ -125,6 +125,8 @@ def merge_users(main_user, other_user, preview=False):
 
     errors = []
     warnings = []
+    if any(course in main_user.get_sorted_courses_responsible_for() for course in other_user.get_sorted_courses_responsible_for()):
```

Comment:
is main_user.get_sorted_courses_responsible_for() called n times here?

Reply:
no, just once because it's independent from `course`.

Reply:
I'd be impressed if python could derive that (function call could modify the object -> next call could be different!) but I'm pretty sure it does not do that :/
```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> def test():
...     print("test called")
...     return [1, 2, 3]
... 
>>> any(number in test() for number in [4, 5, 6, 3])
test called
test called
test called
test called
True
>>> 
```

Reply:
ah, i didn't pay attention and though you meant the second part :)
`other_user.get_sorted_courses_responsible_for()` is only called once.
any suggestions for changes? i actually don't care that much about runtime in user merging.

Reply:
I'd introduce a new variable and store the result of the call there, if that's ok for you. (maybe `courses_where_main_user_is_responsible`?)

Reply:
done